CONTRIBUTING.md

```diff
@@ -1,65 +1,231 @@
-To contribute a fix to a repo, submit a [pull request
-(PR)](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request).
-
-For information on contributing code or API docs to the Flutter SDK, see
-[Contributing to
-Flutter](https://github.com/flutter/flutter/blob/main/CONTRIBUTING.md)
-in the [Flutter SDK](https://github.com/flutter/flutter) repo.
-
-We are also happy to accept your PRs to the
-[Flutter website](https://github.com/flutter/website/) repo,
-even if it's just to fix a typo.
-
-For ways to get involved in the Flutter community or to learn about us,
-visit the [Flutter community](https://flutter.dev/community) page.
-
-# Ways to contribute
-
-We encourage you to reach out and join the conversation.
-
-An easy way to send feedback is to "thumbs up" issues important to you
-in either the issue tracker for the [Flutter SDK and API docs][issues],
-or the [flutter.dev website][doc-issues].
-
-Other ways you can contribute:
-
-* [Ask HOW-TO questions that can be answered with specific solutions][so]
-* [Live chat with Flutter engineers and users][chat]
-* [Discuss Flutter, best practices, app design, and more on our
-   mailing list][mailinglist]
-* [Report bugs and request features][issues]
-* [Report API docs bugs][issues]
-* [Submit PRs to the Flutter SDK][PRs]
-* [Request docs for flutter.dev][doc-issues]
-* [Submit PRs to flutter.dev][doc-PRs]
-* [Follow us on Twitter: @flutterdev](https://twitter.com/flutterdev/)
-* [Read the Flutter Publication on Medium](https://medium.com/flutter)
-* [Sign up to Future UX Studies on Flutter](https://flutter.dev/research-signup)
-* [Join the subreddit to keep up with the latest in the Flutter
-   community][reddit]
-* [Join the Discord to connect with other developers
-   and discuss Flutter][discord]
-
-Happy Fluttering!
-
-[issues]: https://github.com/flutter/flutter/issues
-[PRs]: https://github.com/flutter/flutter/pulls
-[discord]: https://discord.gg/rflutterdev
-[doc-issues]: https://github.com/flutter/website/issues
-[doc-PRs]: https://github.com/flutter/website/pulls
-[so]: https://stackoverflow.com/tags/flutter
-[mailinglist]: https://groups.google.com/d/forum/flutter-dev
-[chat]: https://github.com/flutter/flutter/wiki/Chat
-[reddit]: https://www.reddit.com/r/FlutterDev
+If you would like to contribute to the Flutter website,
+we’re happy to have your help! Anyone can contribute,
+regardless of your writing experience or how you view yourself as a writer,
+developer, or grammar stickler.
+
+## Contributors
+
+Before submitting a PR,
+[search for an issue][search-issue] that covers your concern.
+If you don't find one and your change is significant,
+consider [creating an issue][create-issue]
+that describers the problem and how you would plan to fix it.
+
+When you create a new page or perform a major rewrite of an existing page,
+consider drafting it first in Google Docs to simplify collaboration.
```

Comment:
What do they do with google doc to get the flutter team to review? Do they post the google doc link in an issue they create? Or as a comment if it's an existing issue?